src/main/drivers/pwm_mapping.c

```diff
@@ -1326,43 +1269,43 @@ if (init->useBuzzerP6) {
 
         if (type == MAP_TO_PPM_INPUT) {
 #ifdef REVO
-            if (init->useMultiShot || init->useOneshot || isMotorBrushed(init->motorPwmRate)) {
+            if (init->motorPwmProtocol != MOTOR_PWM_PROTOCOL_1000 || isMotorBrushed(init->motorPwmRate)) {
```

Comment:
isMotorBrushed() is may be not always correct since motor_pwm_rate is also above 500 for the pwm rate protocols?